src/components/StaticAppBar/StaticAppBar.react.js

```diff
@@ -110,7 +110,10 @@ class StaticAppBar extends Component {
         jsonp: 'callback',
         crossDomain: true,
         success: function(data) {
-          let userName = data.settings.userName;
+          let userName = '';
+          if (data.settings && data.settings.userName) {
```

Comment:
Why is a check required on `data.settings.userName`? Doesn't it display the email if the userName is an empty string? Please check. Rest LGTM. Thanks :-)

Reply:
@Akshat-Jain `data.settings.userName` checks if the `userName` parameter is coming or not. If it is blank, then anyways we are initialising `userName` with blank.